api/main.go

```
@@ -0,0 +1,54 @@
+		port, _ := cmd.Flags().GetString("port")
+
+		if port == "" {
+			fmt.Println("Error: --port flag is required")
+			cmd.Usage()
+			os.Exit(1)
+		}
+
+		dbFilePath, _ := cmd.Flags().GetString("db")
+		if dbFilePath == "" {
+			fmt.Println("Error: --db flag is required")
+			cmd.Usage()
```

golangci-lint (1.62.2)

Error return value of `cmd.Usage` is not checked

(errcheck)